plugins/modules/event_info.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Paul Arthur <paul.arthur@flowerysong.com>
+# Copyright: (c) 2019, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'XLAB Steampunk'}
+
+DOCUMENTATION = '''
+module: event_info
+author:
+  - Paul Arthur (@flowerysong)
+  - Aljaz Kosir (@aljazkosir)
+  - Manca Bizjak (@mancabizjak)
+  - Tadej Borovsak (@tadeboro)
+short_description: Lists Sensu events
+description:
+  - For more information, refer to the Sensu documentation at
+    U(https://docs.sensu.io/sensu-go/latest/reference/events/)
+extends_documentation_fragment:
+  - sensu.sensu_go.base
+options:
+  check_name:
+    description:
+      - Limit results to a specific check.
+      - C(entity_name) must also be specified.
+    type: str
+  entity_name:
+    description:
+      - Limit results to a specific entity.
+    type: str
+'''
+
+EXAMPLES = '''
+- name: List Sensu events
+  event_info:
+  register: result
+
+- name: List Sensu events for api.example.com
+  event_info:
+    entity: api.example.com
```

Comment:
```suggestion
    entity_name: api.example.com
```
But to be honest, I think I prefer `entity` and `check` over `entity_name` and `check_name` :grinning: Whatever we decide to go for, let's keep in mind to make example and docs/argument specs consistent :)

Reply:
I couldn't decide between the two, it seems like I forgot to change all the occurrences :grin:. Anyway, I removed the `_name` prefix.